src/main/java/com/cdancy/bitbucket/rest/features/PullRequestApi.java

```diff
@@ -102,4 +102,13 @@ PullRequest reopen(@PathParam("project") String project,
                        @PathParam("repo") String repo,
                        @PathParam("pullRequestId") int pullRequestId,
                        @QueryParam("version") int version);
+
+    @Named("pull-request:changes")
+    @Consumes(MediaType.APPLICATION_JSON)
+    @Path("/{project}/repos/{repo}/pull-requests/{pullRequestId}/changes")
+    @Fallback(PullRequestOnError.class)
+    @GET
+    PagedResponse<Change> changes(@PathParam("project") String project,
```

Comment:
Lets split this out into its own PR as its not related to PullRequestCommentsApi. Also, and for future reference, put all domain objects under their relevant package/folder. The above domain objects `Path`, `Changes`, and `PagedResponse` I would expect to see within `com.cdancy.bitbucket.rest.domain.pullrequest`